pkgs/development/python-modules/pyproj/default.nix

```diff
@@ -1,38 +1,53 @@
-{ lib
-, buildPythonPackage
-, fetchFromGitHub
-, python
-, nose2
+{ lib, buildPythonPackage, fetchPypi, python, pkgs, pythonOlder, substituteAll
+, aenum
 , cython
-, proj ? null
+, proj
+, pytest
+, mock
+, nose2
+, numpy
 }:
 
-buildPythonPackage (rec {
+buildPythonPackage rec {
   pname = "pyproj";
-  version = "unstable-2018-11-13";
+  version = "2.2.1";
 
-  src = fetchFromGitHub {
-    owner = "jswhit";
-    repo = pname;
-    rev = "78540f5ff40da92160f80860416c91ee74b7643c";
-    sha256 = "1vq5smxmpdjxialxxglsfh48wx8kaq9sc5mqqxn4fgv1r5n1m3n9";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0yigcxwmx5cczipf2mpmy2gq1dnl0635yjvjq86ay47j1j5fd2gc";
   };
 
+  # force pyproj to use ${pkgs.proj}
+  patches = [
+    (substituteAll {
+      src = ./001.proj.patch;
+      proj = proj;
```

Comment:
This is proj, not pkgs.proj.

Reply:
oh, right, i guess it was implicitly inheriting it, but it would be best incase there ever is a pythonPackages.proj :)